DigitalLibraryofTennessee/DLTN_XSLT
# Oak ridge transform

**GitHub Issue**: [249](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/249)

* Other Relevant Links (OAI Feeds, DPLA Records, Metadata Mappings, DLTN Documentation, Etc.)
[http://cdm16107.contentdm.oclc.org/oai/oai.php?verb=ListSets](http://cdm16107.contentdm.oclc.org/oai/oai.php?verb=ListSets) 

## What does this Pull Request do?

This PR adds a transform for Oak Ridge Public Library as well as metadata from the two sets they have made available.

## How should this be tested?

Use the transform on one of the sets of metadata and see if DPLA's requirements are met (title, rights, link to object, link to thumbnail, provider). Quickly check to see if object/thumbnail links are correctly formatted and function. Note that Oak Ridge has not turned on IIIF, so these links are not given in the metadata.

## Additional Notes:

Conversation with Julie Forkner and Michael Stallo at Oak Ridge will take place to confirm some of the metadata decisions (for instance, if they are fine with "In Copyright"). We will also be meeting to see if they can split out their dates better in the future (one kind of date for the creation date associated with the object and another kind of date to deal with digitization / donation). For the December ingest we likely won't see any changes in the data.

## Interested parties

@markpbaggett
